Yosemite/Yosemite/Actions/WordPressThemeAction.swift

```diff
@@ -20,4 +20,15 @@ public enum WordPressThemeAction: Action {
     ///     - `result.failure(Error)`: error indicates issues loading themes.
     ///
     case loadCurrentTheme(siteID: Int64, onCompletion: (Result<WordPressTheme, Error>) -> Void)
+
+    /// Installs the given theme ID for a site.
+    /// - `theme`: ID of the theme to be installed.
+    /// - `siteID`: ID of the current site.
+    /// - `onCompletion`: invoked when the theme installation operation finishes.
+    ///     - `result.success(WordPressTheme)`: the installed theme's details.
+    ///     - `result.failure(Error)`: error indicates issues installing themes.
+    ///
+    case installTheme(_ theme: WordPressTheme,
```

Comment:
Same question above about sending only theme ID for installation.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/11430/commits/c070358bc01510ab92f25d10adad96fc32d788b9